internal/db/postgres/pgdump/pgdump.go

```diff
@@ -109,7 +109,21 @@ func (o *Options) GetPgDSN() (string, error) {
 		return o.DbName, nil
 	}
 
-	return fmt.Sprintf("host=%s port=%d user=%s dbname=%s", o.Host, o.Port, o.UserName, o.DbName), nil
+	var parts []string
+	if o.Host != "" {
+		parts = append(parts, fmt.Sprintf("host=%s", o.Host))
+	}
+	if o.Port != 5432 {
+		parts = append(parts, fmt.Sprintf("port=%d", o.Port))
+	}
+	if o.UserName != "" {
+		parts = append(parts, fmt.Sprintf("port=%d", o.Port))
+	}
```

Comment:
There is a bug in this code, it should `user=%d`

Reply:
Thank you for finding this problem. We will publish it in the next release because as was found the variable `PGPASSWORD` is not implemented and you will not be able to connect without a password if was set on DBMS. Unfortunately, this was published by mistake. 

For now, it is better to use `dbname: "postgresql://postgres:example@localhost:54316/original"` instead of usage `host`, `port`, `user` and `dbname`.

Reply:
You're right the PG* variables work fine since they are copying to pg_dump/pg_restore calls and `pgx` driver supports those variables as well. Since this this was published with hurrying we will publish in a few hours another release. That release will fix only this problem. I will rise the issue. Thank you one more time for paying attention to this MR.

Reply:
Thank you for this project, it made my life a lot easier.

Reply:
The raised issue #6

Reply:
We closed this bug in the v0.1.4 version